prow/scripts/cluster-integration/helpers/cleanup-cluster.sh

```diff
@@ -70,7 +70,9 @@ function removeCluster() {
 		date
 		GATEWAY_IP_ADDRESS=$(gcloud compute addresses describe "${CLUSTER_NAME}" --format json --region "${CLOUDSDK_COMPUTE_REGION}" | jq '.address' | tr -d '"')
 		GATEWAY_DNS_FULL_NAME="*.${CLUSTER_NAME}.${DNS_NAME}"
-		IP_ADDRESS=${GATEWAY_IP_ADDRESS} DNS_FULL_NAME=${GATEWAY_DNS_FULL_NAME} "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/delete-dns-record.sh
+		IP_ADDRESS=${GATEWAY_IP_ADDRESS}
+		DNS_FULL_NAME=${GATEWAY_DNS_FULL_NAME}
```

Comment:
splitting this in multiple lines will not work
you'll have to put them back on one line or export `IP_ADDRESS` and `DNS_FULL_NAME`

Reply:
This is the same for all the files below as well

Reply:
I've did small test and this should work.

`[gke_sap-kyma-prow_europe-west3-a_kyma-prow:monitoring]➜  ~ cat test.sh
#! /bin/bash

set -x

ip="192.192.192.19."
name="test"

ip2=${ip}
name2=${name}
./test2.sh ${ip2} ${name2}
[gke_sap-kyma-prow_europe-west3-a_kyma-prow:monitoring]➜  ~ cat test2.sh
#! /bin/bash

set -x

echo ${@}
[gke_sap-kyma-prow_europe-west3-a_kyma-prow:monitoring]➜  ~ ./test.sh
+ ip=192.192.192.19.
+ name=test
+ ip2=192.192.192.19.
+ name2=test
+ ./test2.sh 192.192.192.19. test
+ echo 192.192.192.19. test
192.192.192.19. test
[gke_sap-kyma-prow_europe-west3-a_kyma-prow:monitoring]➜  ~`

Reply:
Yes, it will work, as params are passed as flags, not env variables. Anyway, do we need to rewrite from one variable to another? Why not just pass it as flags?

Reply:
I believe you refer to variables IP_ADDRESS and DNS_FULL_NAME. They were already in place and they value is set by different variables in different places. However I agree that we can just pass as flags. Will rewrite this. Thanks